app/helpers/feed_helper.rb

```diff
@@ -26,10 +26,13 @@ def document_id(record, builder)
     # spec.
     # The interpolation logic is straight out of:
     # http://api.rubyonrails.org/classes/ActionView/Helpers/AtomFeedHelper/AtomFeedBuilder.html#method-i-entry
-    id = record.try(:document) ? record.document.id : record.id
-    return id if record.is_a?(RummagerDocumentPresenter)
+    if record.is_a?(RummagerDocumentPresenter)
+      record.id
+    else
+      id = record.try(:document) ? record.document.id : record.id
```

Comment:
```suggestion
      id = record&.document&.id || record.id
```

I wonder if this might be easier to read.

Reply:
It's definitely shorter. As with all Ruby, I'm not entirely sure what the semantics are to begin with, let alone if they're equivalent, but let's go with it.